libs/shared-atomic-design-components/src/lib/molecules/Editable.scss

```diff
@@ -33,3 +33,17 @@
     fill: $grey-light;
   }
 }
+
+.save__button {
+  cursor: pointer;
+  margin-right: 0.2rem;
+  margin-top: -0.7rem;
+  font-size: $size-5;
+  font-weight: $weight-light;
+  color: $redi-orange-dark;
+}
```

Comment:
Lets remove
```
cursor: pointer;
margin-right: 0.2rem;
font-weight: $weight-light;
```
and add
```
text-transform: capitalize;
letter-spacing: 0.5px;
```
to make it look like in the design:

![image](https://github.com/talent-connect/connect/assets/51786805/d25680c7-0eb3-4bed-9d5f-f63e1f15db0e)